pkg/jobqueue/dbjobqueue/dbjobqueue.go

```diff
@@ -849,13 +867,6 @@ func (q *DBJobQueue) DeleteWorker(workerID uuid.UUID) error {
 	return nil
 }
 
-// connection unifies pgxpool.Conn and pgx.Tx interfaces
-// Some methods don't care whether they run queries on a raw connection,
-// or in a transaction. This interface thus abstracts this concept.
-type connection interface {
-	Query(ctx context.Context, sql string, args ...interface{}) (pgx.Rows, error)
-}
-
```

Comment:
Ha, little guy. I thing it would not hurt search-n-replacing all connection arguments to use this interface so next time someone copy-paste a method it is already there and ready :-) Great find.